### PR TITLE
Adding portuguese strings.xml

### PR DESCRIPTION
**Pull request by theolm**
This PR adds Brazilian Portuguese strings to the app.

ps. I noticed some hardcoded strings in the app.

### CODE REVIEW

File: app/src/main/res/values-pt/strings.xml [hunk 1]
```diff
@@ -0,0 +1,221 @@
+<resources>
+    <!-- Outras Strings -->
```
Comment (starry-shivam) [inline app/src/main/res/values-pt/strings.xml:2]: Please don't translate these comments; they're there to help me and other contributors identify which strings belong where. So when I need to extend an existing feature, I can quickly find and update the strings related to that. If you want to add translations anyway, then do it like this:

`<!-- original title | translated title -->`

**Review by starry-shivam — CHANGES_REQUESTED** [review]
Thanks for your contribution! Could you point out which strings you're referring to as hardcoded?

File: app/src/main/res/values-pt/strings.xml (hunk 1 above)
Comment (theolm) [reply app/src/main/res/values-pt/strings.xml]: Done.

Comment (theolm) [reply app/src/main/res/values-pt/strings.xml]: It's not exactly hardcoded strings, I should have explained better.

You are using the `Enum.name` in several places this practice breaks the localization and the end result is the same as a hardcoded string.

Example:
GoalPriority.High.name

**Review by starry-shivam — APPROVED** [review]
Thanks again.

**Pull request merged**